packages/dds/tree/README.md

```diff
@@ -272,6 +272,19 @@ graph LR;
     Indexes--"delta (for apps that want deltas)"-->app
 ```
 
+### Schema Evolvability
+
+Over time, application authors may want to change the schema for their documents.
+For example, they might want to add support for a new application feature or represent existing content in some new way.
+
+Before doing so, application authors must consider compatibility constraints within their ecosystem.
+Most ecosystems don't have a way to ensure all users within a collaborative session are using the same code version.
```

Comment:
It might be worth including the simpler non-collab case here. Maybe:

```suggestion
Most ecosystems don't have a way to ensure all documents the application may open are using the new schema or even that all users within a collaborative session are using the same code version.
```